zap/src/main/java/org/zaproxy/zap/extension/pscan/PassiveScanTask.java

```diff
@@ -204,16 +204,18 @@ public void run() {
                 }
             } else {
                 logger.error(
-                        "Parser failed on record {} from History table", href.getHistoryId(), e);
+                        "Parser failed on record {} from History table (perhaps it was deleted between when it was enqueued and now)",
```

Comment:
IMO it could just check if the record still exists or not, if it doesn't not worth log anything.

Reply:
You mean here or just before processing it?

Reply:
Here.

Reply:
I think I'm overthinking this, what call should I be making to see if the record still exists?

Reply:
`TableHistory#read` it returns null if it no longer exists.